render/allocator.c

```diff
@@ -29,10 +28,7 @@ struct wlr_allocator *allocator_autocreate_with_drm_fd(
 	if ((backend_caps & gbm_caps) && (renderer_caps & gbm_caps)
 			&& drm_fd != -1) {
 		wlr_log(WLR_DEBUG, "Trying to create gbm allocator");
-		int fd = fcntl(drm_fd, F_DUPFD_CLOEXEC, 0);
-		if (fd < 0) {
-			wlr_log(WLR_ERROR, "fcntl(F_DUPFD_CLOEXEC) failed");
-		} else if ((alloc = wlr_gbm_allocator_create(fd)) != NULL) {
+		if ((alloc = wlr_gbm_allocator_create(drm_fd)) != NULL) {
```

Comment:
This breaks the GBM allocator temporarily, it's fixed in the next commit. This will make git-bisect operations more difficult, instead we should move this hunk to the next commit.